api/system-certificate/read

```diff
@@ -61,6 +65,24 @@ if ($cmd eq 'list') {
     set_default('CommonName','NethServer');
     set_default('EmailAddress',"");
     set_default('SubjectAltName',$db->get_type('DomainName'));
+
+    # Output all know  domains/FQDN of the server
+    my @domains = ();
+
+    foreach my $d ($dbd->get_all()) {
+        my $domain = $d->key;
+        push @domains, $domain;
+    }
+    foreach my $h ($dbh->get_all_by_prop('type'=>'self')) {
+        my $domain = $h->key;
+        push @domains, $domain;
+    }
+
+    push @domains, $db->get('SystemName').'.'.$db->get('DomainName'),$db->get('DomainName');
+    my $domains = join (',',do { my %seen; grep { !$seen{$_}++ } @domains });
+
+    $out->{'KnowDomains'} = $domains;
```

Comment:
Please reword it to a more correct form:
```suggestion
    $out->{'KnownDomains'} = $domains;
```

Reply:
done